docs/installation.rst

```diff
@@ -29,9 +29,11 @@ pip - from GitHub
 Additional Prerequisites
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. `CMake <https://cmake.org/>`__ version 3.18 or later
-2. `pyTorch <https://pytorch.org/>`__ with GPU support
-3. `Ninja <https://ninja-build.org/>`__
+1. [Not needed for JAX] `CMake <https://cmake.org/>`__ version 3.18 or later.
+2. [Optional] `Ninja <https://ninja-build.org/>`__: `pip install ninja`.
+3. [For pyTorch support] `pyTorch <https://pytorch.org/>`__ with GPU support.
+3. [For JAX support] `JAX <https://github.com/google/jax/>`__ with GPU support, version >= 0.4.7.
+3. [For JAX support] `pybind11`: `pip install pybind11`.
```

Comment:
```suggestion
1. `CMake <https://cmake.org/>`__ version 3.18 or later.
2. `pybind11 <https://github.com/pybind/pybind11>`__ : `pip install pybind11`.
3. [Optional] `Ninja <https://ninja-build.org/>`__: `pip install ninja`.
4. [For PyTorch support] `PyTorch <https://pytorch.org/>`__ with GPU support.
5. [For JAX support] `JAX <https://github.com/google/jax/>`__ with GPU support, version >= 0.4.7.
6. [For TensorFlow support] `TensorFlow <https://www.tensorflow.org/>`__ with GPU support.
```
CMake and pybind11 are used to build the core library, so I don't think they should be treated as optional . The reason we don't need to install pybind11 in the PyTorch build is because it's a PyTorch dependency.

Reply:
This isn't an optional dependency. There isn't an optional section.
To prevent confusion, I just added a commit that move the optional one at the end of the list.

Reply:
updated to reflect that.